### ISSUE TITLE
Make it easy to navigate

### ISSUE THREAD

**Issue by drdwitte** [RubenVerborgh/WebFundamentals#11]
Due to the many links it could be possible that you accidently skip part of a lecture. Maybe it would be good to have a small window to mark your progress: page number, % explored,... (not always evident if people would explore the slides in a nonlinear way)

**Comment by RubenVerborgh** [comment]
Indeed, saving progress would be good! I purposely avoided slide numbers (because they distract during a presentation, and—for me—don't make sense in a non-linear context), but that is the downside to that.

Maybe we good even have additional features here, such as marking whether or not a slide has been studied/understood etc.
